Drv/LinuxUartDriver/LinuxUartDriver.cpp

```diff
@@ -346,7 +346,7 @@ void LinuxUartDriver ::serialReadTaskEntry(void* ptr) {
     Drv::RecvStatus status = RecvStatus::RECV_ERROR;  // added by m.chase 03.06.2017
     LinuxUartDriver* comp = reinterpret_cast<LinuxUartDriver*>(ptr);
     while (!comp->m_quitReadThread) {
-        Fw::Buffer buff = comp->allocate_out(0, comp->m_allocationSize);
+        Fw::Buffer buff = comp->allocate_out(0, static_cast<U32>(comp->m_allocationSize));
```

Comment:
m_allocationSize needs to be made into correct type.  Since it is a buffer allocation it should be U32.